src/config/dynamic-theme-fixes.config

```diff
@@ -8276,6 +8276,21 @@ body {
 
 ================================
 
+*.suckless.org
```

Comment:
Change it to the right place and I meant the *.suckless.org is redunant.

Reply:
50fb573 Done